yelp_beans/logic/metrics.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from collections import defaultdict
+
+from yelp_beans.logic.meeting_spec import get_specs_for_current_week
+from yelp_beans.logic.meeting_spec import get_users_from_spec
+from yelp_beans.models import MeetingSubscription
+from yelp_beans.models import User
+
+
+def get_subscribed_users():
+    users = User.query().fetch()
+    subscriptions = MeetingSubscription.query().fetch()
+
+    metrics = defaultdict(set)
+    for subscription in subscriptions:
+        metrics[subscription.key.urlsafe()] = []
```

Comment:
There are two types of subscriptions.  One with users and one without.  The second for-loop doesn't populate subscriptions without users because it iterates through users.  The for-loop you commented on specifically ensures that even subscriptions with no users are represented in the data structure.  

This isn't too straightforward, maybe it should have a comment?